### PR TITLE
add some scala doc and python doc

### PR DESCRIPTION
**Pull request by zhangxiaoli73**
## What changes were proposed in this pull request?

add scala doc and python doc

### CODE REVIEW

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Tanh.scala [hunk 1]
```diff
@@ -23,6 +23,10 @@ import com.intel.analytics.bigdl.tensor._
 
 import scala.reflect.ClassTag
 
+/**
+ * Applies the Tanh function element-wise to the input Tensor,
```
Comment (qiuxin2012) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Tanh.scala]: Please specify the definition of Tanh

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/dataset/image/LocalImgReader.scala [hunk 2]
```diff
@@ -59,6 +59,13 @@ class LocalScaleImgReader private[dataset](scaleTo: Int, normalize: Float)
   }
 }
 
+/**
+ * Read BGR images from given paths. After read the image, it will resize the images to the given
```
Comment (qiuxin2012) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/dataset/image/LocalImgReader.scala]: I think it's a local reader.

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Tanh.scala (hunk 1 above)
Comment (zhangxiaoli73) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Tanh.scala]: ok, changed

File: spark/dl/src/main/scala/com/intel/analytics/bigdl/dataset/image/LocalImgReader.scala (hunk 2 above)
Comment (zhangxiaoli73) [inline spark/dl/src/main/scala/com/intel/analytics/bigdl/dataset/image/LocalImgReader.scala]: ok, changed

**Pull request merged**